baseline/client/tests/n-back.test.js

```diff
@@ -241,9 +241,18 @@ describe("n-back", () => {
 
     it("n-back plugin trials are succeeded by rests", () => {
         const flatTimeline = (new NBack(1)).getTimeline();
```

Comment:
Why is this flatTimeline? Did you mean to wrap it in flattenTimeline?

Reply:
Oops! That's right, I meant to wrap it but I didn't.